projects/igniteui-angular/src/lib/tabs/tabs.component.ts

```diff
@@ -365,9 +387,13 @@ export class IgxTabsComponent implements IgxTabsBase, AfterViewInit, OnDestroy {
                         const newTab = this.tabs.toArray()[this._selectedIndex];
                         this.transformContentAnimation(newTab, 0);
                     }
+                    this.updateScrollButtons();
+
                 });
 
                 this._resizeObserver.observe(this.tabsContainer.nativeElement);
+                this._resizeObserver.observe(this.headerContainer.nativeElement);
```

Comment:
The widths of the `tabsContainer` and `headerContainer` are equal, so I think we could listen only for changes of the `tabsContainer` and `viewPort`.